docs/docs/api/promise.coroutine.md

```diff
@@ -13,7 +13,9 @@ title: Promise.coroutine
 Promise.coroutine(GeneratorFunction(...arguments) generatorFunction, Object options) -> function
 ```
 
-Returns a function that can use `yield` to yield promises. Control is returned back to the generator when the yielded promise settles. This can lead to less verbose code when doing lots of sequential async calls with minimal processing in between. Requires node.js 0.12+, io.js 1.0+ or Google Chrome 40+.
+In a typical use case, the parameter `generatorFunction` should be a `GeneratorFunction` producing a generator that yields one or more `Promise`s. (Also see the Tips section below.)
+
+This method returns a function that returns a `Promise`(TODO: Elaborate this). When the returned function (e.g. the `ping` function in the example below) is called, it will start the generator (by calling its `next()`) and return immediately. Control is returned back to the generator when each yielded promise settles. This can lead to less verbose code when doing lots of sequential async calls with minimal processing in between. Requires node.js 0.12+, io.js 1.0+ or Google Chrome 40+.
```

Comment:
> Requires node.js 0.12+, io.js 1.0+ or Google Chrome 40+.

Could this be expanded, perhaps?  It'd certainly be good to know what IE support looks like, and whether this can be supported by a polyfill, or if there's a more fundamental limitation.  (In the comments on the page, I see someone who says that a polyfill enables it for IE10, but not IE9, but I see no confirmation of that)

I can dig into this, but I'm guessing someone more knowledgeable might know off-hand.